src/corner.rs

```diff
@@ -107,11 +110,29 @@ impl Corner {
             .unwrap_or(true)
     }
 
-    fn execute_command(&self) -> Result<()> {
-        if let Some(binary) = self.config.command.first() {
+    fn execute_enter_command(&self) -> Result<()> {
+        if let Some(binary) = self.config.enter_command.first() {
             let args = self
                 .config
-                .command
+                .enter_command
+                .iter()
+                .enumerate()
+                .filter(|(index, _)| index > 0.borrow())
+                .map(|(_, value)| value)
+                .collect::<Vec<_>>();
+            info!("executing command: {} {:?}", binary, args);
+            let output = Command::new(binary).args(args).output()?;
+            info!("output received: {:?}", output);
+        }
+
+        Ok(())
+    }
+
+    fn execute_exit_command(&self) -> Result<()> {
+        if let Some(binary) = self.config.exit_command.first() {
+            let args = self
+                .config
+                .exit_command
```

Comment:
Could there be 1 method called `execute_command` that allows a `Vec<String>` as an input so each of these could call that with the config value so the logic doesn't need to be replicated?